opennlp-tools/src/main/java/opennlp/tools/ml/model/MaxentModel.java

```diff
@@ -33,7 +33,7 @@
    *         outcomes, all of which sum to 1.
    *
    **/
-  public double[] eval(String[] context);
+  double[] eval(String[] context);
```

Comment:
This file should not be changed with the commit. There are many cases where we have a redundant public modifier. If there is a desire to change that, it would be best to use a re-factoring tool and do this once for the entire code base.